tests/e2e.bats

```diff
@@ -175,3 +175,32 @@
   [ "$status" -eq 0 ]
   [[ "$output" == "dummy_bpf.o:[classifier]" ]]
 }
+
+# Test case for validating multiple devices allocated to the same pod.
+@test "2 dummy interfaces with IP addresses ResourceClaimTemplate" {
+  docker exec "$CLUSTER_NAME"-worker bash -c "ip link add dummy6 type dummy"
+  docker exec "$CLUSTER_NAME"-worker bash -c "ip link set up dev dummy6"
+  docker exec "$CLUSTER_NAME"-worker bash -c "ip addr add 169.254.169.13/32 dev dummy6"
+
+  docker exec "$CLUSTER_NAME"-worker bash -c "ip link add dummy7 type dummy"
+  docker exec "$CLUSTER_NAME"-worker bash -c "ip link set up dev dummy7"
+  docker exec "$CLUSTER_NAME"-worker bash -c "ip addr add 169.254.169.14/32 dev dummy7"
+
+  kubectl apply -f "$BATS_TEST_DIRNAME"/../examples/deviceclass.yaml
+  kubectl apply -f "$BATS_TEST_DIRNAME"/../examples/resourceclaimtemplate_double.yaml
+  kubectl wait --timeout=30s --for=condition=ready pods -l app=MyApp
+  POD_NAME=$(kubectl get pods -l app=MyApp -o name)
+  run kubectl exec $POD_NAME -- ip addr show dummy6
+  [ "$status" -eq 0 ]
```

Comment:
If you rebase, you'll find a different pattern being used to perform assertions in bats tests. Please use those instead since they log both the got/want ouptuts.

Reply:
Gotcha! Looks like this is failing at the moment anyways so I need to go back.

Reply:
Done!